src/test/java/com/ays/admin_user/service/impl/AdminUserRegisterServiceImplTest.java

```diff
@@ -102,7 +102,7 @@ void givenValidAdminUserRegisterRequest_thenAdminUserRegistered() {
         Mockito.verify(adminUserRepository, Mockito.times(1))
                 .existsByUsername(Mockito.anyString());
         Mockito.verify(adminUserRepository, Mockito.times(1))
-                .existsByCountryCodeAndLineNumber(Mockito.anyLong(), Mockito.anyLong());
+                .existsByCountryCodeAndLineNumber(Mockito.anyString(), Mockito.anyString());
         Mockito.verify(adminUserRepository, Mockito.times(1))
                 .save(Mockito.any(AdminUserEntity.class));
         Mockito.verify(adminUserRegisterVerificationRepository, Mockito.times(1))
```

Comment:
Bu sınıf içerisindeki geliştirmelere ihtiyaç kalmadığı için geri alabilir miyiz?